### ISSUE TITLE
Add Accessibility Statement & do accessibility review 

### ISSUE THREAD

**Issue by mgifford** [taigaio/taiga-front#54]
**Please describe the problem / need you are trying to solve.**
I'd like to know if teams with people with disabilities are able to use this kanban tools. Right now I have no idea if this tool complies with WCAG 2.1 AA or is usable by people with disabilites. 

**Describe the feature or the improvement you'd like and what are you trying to achieve.**
I'd like to see an accessibility statement & maybe a VPAT
https://www.w3.org/WAI/planning/statements/
https://www.itic.org/policy/accessibility/vpat

1/5 people has a disability. Many are invisible. People's abilities tend to deteriorate over time. Accessibility needs to be built in.

**Comment by antgel** [comment]
Orthogonally related to #61 .

**Comment by diacritica** [comment]
Hi Mike!

The Taiga team has been always very sensitive to accessibility, not only because one Taiga co-founder suffers from visual impairment (one of the many disabilities there) but also for the pure sake of making Taiga more universally inclusive. Some years ago, there was even an attempt to develop a terminal Taiga client.

See 

![](https://bameda.github.io/slides/2015/pycones_2015/hacking_the_taiga/img/community-examples/taiga-ncurses-1.png)

![](https://bameda.github.io/slides/2015/pycones_2015/hacking_the_taiga/img/community-examples/taiga-ncurses-2.png)

More recently, though, the team has decided to have the Taiga web client itself accessible. This is part of the next Taiga major release due 2022. The way this will be done is to agree on a set of accessibility guidelines (publicly available) and then apply those guidelines to all end user facing features. This might be implemented as a user-story child task labelled accordingly, but we are still discussing the best approach.

Both designers and coders in the Taiga team consider this a very important aspect of the future Taiga release and the product and community as a whole. And while we fear there will some aspects that might not be properly addressed, we will pursue as much as possible in this aspect. 

Since we don't plan to develop the next major Taiga release "in the shadows", some aspects of this should be disclosed and shared perhaps by the end of the year. We would love to have you engaged at that point so we have high quality feedback.

So, we're deeply sorry to answer “not at this time” but this is part of our future work. At the moment, Taiga6 is leaning towards accessibility but it's limited, mostly around visual impairment, which designers took into account in the past. But we don't feel it's too solid to deserve an official statement at this time. We hope this clarifies the situation for you and the Taiga community as a whole.

**Comment by antgel** [comment]
Thanks for responding. I think it's a bit of a contradiction:

> we don't plan to develop the next major Taiga release "in the shadows"

and

> some aspects of this should be disclosed and shared perhaps by the end of the year

Some aspects? Perhaps? By the end of the year?

I don't think you understand how important this is. You're treating accessibility as an afterthought rather than an integral part of your process. In the worst case, many jurisdictions, not being compliant with say, WCAG 2.1 AA, can result in legal action.

**Comment by diacritica** [comment]
Hi Antony

Perhaps I wrote the piece in a hurry, sorry about that. I reiterate that for the next major release, accessibility is not an afterthought, quite the contrary. There will be accessibility guidelines for the team to follow on each existing or new feature so it will be an integral part of the process. What I meant by "disclosed by the end of the year" is that we will share an early "shot" of what that next major release of Taiga will look like. That piece of software, even if it's not entirely functional, could be subject to feedback (including accessibility) Guidelines, partial work, etc, everything under our open source development, will always be available, as has been since 2015.

**Comment by mgifford** [comment]
@diacritica thanks for this. Would love to see an accessibility statement that makes this goal clear for everyone who is considering using this tool. I think it would really set you apart from the crowd. 

Please do what you can to communicate more about your work to become more inclusive. Goal isn't perfection, but being more accessible today than you were yesterday.

**Comment by suntorytimed** [comment]
@diacritica that terminal Taiga client looks awesome btw.
